1.8/usage/cli/install.md

```diff
@@ -79,20 +120,19 @@ The easiest way to install the DC/OS CLI is by clicking the signpost icon in the
     Get detailed command description with 'dcos <command> --help'.
     ```
 
-
 # <a name="windows"></a>Installing on Windows
 
 #### Prerequisites
 
 *   A system external to your DC/OS cluster onto which you will install the CLI
 *   Network access from the external system to your DC/OS cluster
-*   Windows PowerShell: Installed by default on Windows 7 and later
+*   A command-line environment, such as Windows Powershell: Installed by default on Windows 7 and later
 *   Disable any security or antivirus software before beginning the installation.
 
 
-1.  Run Powershell as Administrator.
+1.  Run command-line environment as Administrator.
 
-1.  Download and install the DC/OS CLI executable ([dcos.exe](https://downloads.dcos.io/binaries/cli/windows/x86-64/0.4.10/dcos.exe)).
+1.  Download the DC/OS CLI executable to your local directory ([dcos.exe](https://downloads.dcos.io/binaries/cli/windows/x86-64/0.4.10/dcos.exe)).
```

Comment:
Adding the comment here but it applies to the "Authenticate..." section below. Keep `dcos auth login` but remove the section that talks about asking for password. Instead ask them to follow the instructions printed by the DC/OS CLI.